### PR TITLE
Convert toml to yaml, update HTML5 tutorials, remove old docs

### PR DESCRIPTION
**Pull request by MasterPtato**
Fixes WEB-108
Fixes WEB-61
Fixes WEB-109
Fixes WEB-110

### CODE REVIEW

**Comment by MasterPtato** [comment]
Current dependencies on/for this PR:
* `main`
  * **PR #90** <a href="https://app.graphite.dev/github/pr/rivet-gg/site/90?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/rivet-gg/site/90?utm_source=stack-comment).

File: package.json [hunk 1]
```diff
@@ -38,7 +38,7 @@
     "framer-motion": "7.8.1",
     "mdast-util-to-string": "^3.2.0",
     "mdx-annotations": "^0.1.1",
-    "next": "14.1.0",
+    "next": "13.4.2",
```
[the remark this answers is not in the document]
Comment (MasterPtato) [reply package.json:41]: Forgot to mention you about this, I was getting all sorts of weird errors from the upgrade. Was there any specific reason to update these packages?

**Review by NathanFlurry — APPROVED** [review]
lgtm. cf pages is failing.

File: package.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (MasterPtato) [reply package.json]: reverted to fix cf deploy error

**Pull request merged**